src/test/java/study/string/CharAtTest.java

```diff
@@ -0,0 +1,26 @@
+package study.string;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class CharAtTest {
+    private static final String ABC ="abc";
+
+    @ParameterizedTest
+    @DisplayName("추출 테스트")
+    @CsvSource(value = {"0,a","1,b","2,c"})
```

Comment:
```suggestion
    @CsvSource(value = {"0,a", "1,b", "2,c"})
```
이렇게 컨벤션도 맞춰보면 좋을 것 같아요
다음 미션부터 차근차근 맞춰보는 것을 목표로 해보시죠!

<img width="934" alt="image" src="https://github.com/user-attachments/assets/c7565187-a47e-47e3-a089-39093abbb6bd">